# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/aspell/gcc33-support.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- aspell-0.50.3/prog/checker_string.cpp.orig	2003-08-13 16:45:30.000000000 +0200
+++ aspell-0.50.3/prog/checker_string.cpp	2003-08-13 16:53:13.000000000 +0200
@@ -9,6 +9,9 @@
 #include "document_checker.hpp"
 #include "copy_ptr-t.hpp"
 #include "asc_ctype.hpp"
+#if __GNUC__ == 3
+#include <cassert>
+#endif
 
 static int get_line(FILE * in, CharVector & d)
 {
